monkey/monkey_island/cc/resources/auth/auth_user.py

```diff
@@ -0,0 +1,11 @@
+__author__ = 'itay.mizeretz'
+
+
+class User(object):
+    def __init__(self, user_id, username, secret):
+        self.id = user_id
```

Comment:
Are we ever using the UserId?

Reply:
No. I presumed that JWT interface requires it and that's why it was added in the first place. But original creator might've been wrong, I'll check